configuration.md

```diff
@@ -0,0 +1,72 @@
+
+# Configuration properties
+
+## Configuration object model
+
+```mermaid
+classDiagram
+  GatewayConfigProperties *-- HeaderMappings : defaultHeaders
+  GatewayConfigProperties *-- "0..*" RoleBasedAccessRule : globalAccessRules
+  GatewayConfigProperties *-- "0..*" Service
+  Service *-- HeaderMappings : headers
+  Service *-- "0..*" RoleBasedAccessRule : accessRules
+  class GatewayConfigProperties{
+    Map<String, Service> services
+  }
+  class HeaderMappings{
+    boolean proxy
+    boolean username
+    boolean roles
+    boolean org
+    boolean orgname
+    boolean email
+    boolean firstname
+    boolean lastname
+    boolean tel
+    boolean jsonUser
+    boolean jsonOrganization
```

Comment:
I don't get what is the purposes of these boolean yet (maybe later reading the rest of the PR)